submissions/BlueLamantine/memory-pair-game/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hairy Pawtter</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="css/map.css" />
```

Comment:
Avoid using so many style files on some page. It can drastically reduce page loading time.
I'd recommend using Webpack for your projects in order to use modules in your app.
You have not to do it in current task, but keep this thing in mind for future projects

Reply:
ok,  I'll keep it in my mind!